modules/EnsEMBL/Draw/GlyphSet/genetree.pm

```diff
@@ -46,9 +46,10 @@ sub _init {
 ## EG
   my $highlight_annotations   = $self->{highlights}->[8]; # EG
 ##
-  my $tree          = $self->{'container'};
-  my $Config        = $self->{'config'};
-  my $bitmap_width = $Config->image_width(); 
+  my $tree           = $self->{'container'};
+  my $Config         = $self->{'config'};
+  my $highlight_gene = $Config->{_parameters}->{highlight_gene};
```

Comment:
Can you use this instead to get the parameter:
$Config->get_parameter('highlight_gene'); 
Let me know if it works

Reply:
It worked. Thanks 😄